packages/dashboard-frontend/src/components/ImportFromGit/AdvancedOptions/CpuLimitField/index.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2018-2024 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { FormGroup, Slider } from '@patternfly/react-core';
+import React from 'react';
+
+const STEP = 1;
+
+const steps = [
+  { value: 0, label: 'default' },
```

Comment:
It is impossible because of the interface for steps:
```ts
interface SliderStepObject {
    /** Flag to hide the label. */
    isLabelHidden?: boolean;
    /** The display label for the step value. This is also used for the aria-valuetext attribute. */
    label: string;
    /** Value of the step. This value is a percentage of the slider where the tick is drawn. */
    value: number;
}
```